app/controllers/spina/conferences/primer_theme/journal/issues_controller.rb

```diff
@@ -13,7 +13,8 @@ class IssuesController < ApplicationController
           def index
             # having multiple journals is not currently allowed anyway
             @issues = Admin::Journal::Issue.sorted_desc
-            @latest_issue = @issues.find_by 'date <= ?', Time.zone.today
+            @issues = @issues.where('date <= ?', Time.zone.today) unless current_spina_user.present?
```

Comment:
Why is issues unset in this case?

Reply:
maybe this would be more semantic as an if/else - issues is set in both cases. one moment